nilearn/decoding/space_net_solvers.py

```diff
@@ -9,25 +9,26 @@
 # License: simplified BSD
 
 from math import sqrt
+
 import numpy as np
 from nilearn.masking import _unmask_from_to_3d_array
+
+from .fista import mfista
+from .objective_functions import _div, _gradient, _gradient_id
+from .objective_functions import _logistic as _logistic_loss
```

Comment:
I don't understand the logic here : Why is this import segmented in such a way ?

Reply:
Aliased imports are treated separately by default by isort.

But this can be changed in the settings.

https://github.com/PyCQA/isort/wiki/isort-Settings#full-reference-of-isort-settings

Reply:
Makes sense, but then we should put all the others on the same line.

Reply:
Since manual tweaks defeats the purpose of using things like isort, I will try tweaking the config instead.

Reply:
Anyway, this is a tiny thing, we should not worry about that ;-)

Reply:
no worries
I'd rather spend a bit more time on the config now and not touch anymore in the future.

See commit https://github.com/nilearn/nilearn/pull/3538/commits/554d92a90f56f3640d49ffe9d9b27dce5779b23d for a how the new config affect things.